source/fuzz/fuzzer_context.cpp

```diff
@@ -126,6 +126,8 @@ const std::pair<uint32_t, uint32_t> kChanceOfSwappingConditionalBranchOperands =
     {10, 70};
 const std::pair<uint32_t, uint32_t> kChanceOfTogglingAccessChainInstruction = {
     20, 90};
+const std::pair<uint32_t, uint32_t> kChanceOfWrappingRegionInSelection = {10,
+                                                                          90};
```

Comment:
My inclination is to make the upper probability a bit lower, e.g. 70.  This transformation is cool but a little bit strange so I don't think we want to apply it too aggressively.

Reply:
Fixed in 2613291.